meta_test.go

```diff
@@ -1,216 +1,223 @@
 package main
 
 import (
-	"bytes"
+	"github.com/screwdriver-cd/meta-cli/internal/fetch"
```

Comment:
please reorder imports - here's what I'm getting after pressing "Save" in my editor (with `goimports` integrated):
```
import (
	"io"
	"io/ioutil"
	"net/http"
	"net/http/httptest"
	"os"
	"path/filepath"
	"testing"

	"github.com/screwdriver-cd/meta-cli/internal/fetch"
	"github.com/stretchr/testify/mock"
	"github.com/stretchr/testify/require"
	"github.com/stretchr/testify/suite"
	"github.com/termie/go-shutil"
)
```